conanfile.py

```diff
@@ -44,27 +41,42 @@ def requirements(self):
             self.requires("gtest/1.14.0")
             self.requires("boost/1.84.0")
         if self.options.build_unbundled: #each componenet is built independently 
-            self.requires("up-cpp/0.1.1-dev")
-            if self.options.zenoh_package:
-                self.requires("zenohc/cci.20240213")
-        if self.options.build_cross_compiling :
+            self.requires("up-cpp/x.y.z")
             self.requires("zenohc/cci.20240213")
```

Comment:
The change in structure above means that the default is now to include this line, the equivalent of `build_unbundled=True` and `zenoh_package=True` (before, the latter was false). However, zenohc is not available as a package in the conan repositories. The old default behavior was to use up-cpp from the local conan cache and zenoh-c from the system libs (`build_unbundled=True` and `zenoh_package=False`). The only way to get past the `conan install` step now is to set `build_unbundled=False`, but that causes the cmake stage to not work.

Two items need to be addressed:
- The reference to a non-existent conan package should be removed
- README.md should be updated with instructions based on the change in build options such that a developer with a default Ubuntu 22.04 machine could build this library and run the tests.

Reply:
build_unbundled=True should be default in general flow